storybook/pages/KeychainPage.qml

```diff
@@ -233,15 +130,15 @@ Page {
         target: loader.item
 
         function onSaveCredentialRequestCompleted(status) {
-            logs.logEvent("SaveCredentials", ["status"], [status])
+            logs.logEvent("SaveCredentials", ["status"], arguments)
```

Comment:
what's `arguments`? we only have `status` variable here 🤔

Reply:
js :grin: `arguments` means array of all arguments provided to the function